test/pruning/test_pytorch_pruning_cyy.py

```diff
@@ -0,0 +1,210 @@
+import os
```

Comment:
what's the meaning of _cyy file name postfix?

Reply:
done